src/eglext.rs

```diff
@@ -42,18 +45,36 @@ extern {}
 pub fn CreateImageKHR(dpy: EGLDisplay, context: EGLContext, target: EGLenum,
                       buffer: EGLClientBuffer, attrib_list: *const EGLint) -> EGLImageKHR {
     unsafe {
+        let name = ffi::CString::new("eglCreateImageKHR").unwrap().as_ptr();
+
+        let addr = eglGetProcAddress(name as *const _);
+
+        if addr == ptr::null() {
+            panic!("Unable to find an entry point for eglCreateImageKHR");
+        }
+
+        let eglCreateImageKHR: extern "C" fn(dpy: EGLDisplay, context: EGLContext, target: EGLenum,
+                                             buffer: EGLClientBuffer, attrib_list: *const EGLint) -> EGLImageKHR = mem::transmute(addr);
         return eglCreateImageKHR(dpy, context, target, buffer, attrib_list);
     }
 }
 
 pub fn DestroyImageKHR(dpy: EGLDisplay, image: EGLImageKHR) -> EGLBoolean {
     unsafe {
+        let name = ffi::CString::new("eglDestroyImageKHR").unwrap().as_ptr();
```

Comment:
If I hold onto the result of the unwrap() and .as_ptr() inline with the getprocaddr call, I should be fine, right?

/me notes the irony present in introducing a UAF in uncalled code in an unused module, in an effort to avoid bumping semver by just removing the obviously-bogus code.